gen/dcos-config.yaml

```diff
@@ -701,6 +701,20 @@ package:
     content: |
         {
           "HTTPEndpoints": [
+            {% switch dcos_overlay_enable %}
+             {% case "true" %}
```

Comment:
The spacing seems weird here. Is a single space indent intended?

Reply:
To match the style of the rest of this template, the `switch` and `case` statements here, and the `case` and `endswitch` below, should be completely de-indented to the beginning of their respective lines, with no preceding whitespace.